src/test/java/org/cactoos/scalar/BoolOfTest.java

```diff
@@ -37,28 +37,28 @@
 
     @Test
     public void trueTest() throws Exception {
-        MatcherAssert.assertThat(
-            "Can't parse 'true' string",
+        new Assertion<>(
+            "Must be parsed string 'true'",
             new BoolOf("true").value(),
-            Matchers.equalTo(true)
-        );
+            new IsEqual<>(true)
```

Comment:
@marceloamadeu can you replace all those `new BoofOf(xxx).value()` with `new BoofOf(xxx)` and use `ScalarHasValue` as the matcher?